node/node-core/src/eth/receipt.rs

```diff
@@ -0,0 +1,161 @@
+use crate::eth::utils::enveloped;
+use ethers_core::{
+    types::{Address, Bloom, Bytes, H256, U256},
+    utils::{
+        rlp,
+        rlp::{Decodable, DecoderError, Encodable, Rlp, RlpStream},
+    },
+};
+use foundry_evm::revm;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+pub struct Log {
+    pub address: Address,
+    pub topics: Vec<H256>,
+    pub data: Bytes,
+}
+
+impl From<revm::Log> for Log {
+    fn from(log: revm::Log) -> Self {
+        let revm::Log { address, topics, data } = log;
+        Log { address, topics, data: data.into() }
+    }
+}
```

Comment:
Wonder if we can avoid this type altogether if we just PR the needed type changes to revm?

Reply:
different bytes Types I believe, but can have a look